apps/mobile/src/features/accounts/balances/hooks/mempool/use-mempool-txs-balance.ts

```diff
@@ -0,0 +1,44 @@
+import type {
+  MempoolTokenTransferTransaction,
+  MempoolTransaction,
+  Transaction,
+} from '@stacks/stacks-blockchain-api-types';
+
+import { createMoney, sumNumbers } from '@leather.io/utils';
+
+import { useStacksConfirmedTransactions } from '../stacks/use-stacks-confirmed-transactions';
+import { useStacksPendingTransactions } from '../stacks/use-stacks-pending-transactions';
+
+interface CalculatePendingTxsMoneyBalanceArgs {
+  txs: MempoolTokenTransferTransaction[];
+}
+function calculatePendingTxsMoneyBalance({ txs }: CalculatePendingTxsMoneyBalanceArgs) {
+  const tokenTransferTxsBalance = sumNumbers(txs.map(tx => Number(tx.token_transfer.amount)));
+  const pendingTxsFeesBalance = sumNumbers(txs.map(tx => Number(tx.fee_rate)));
+
+  return createMoney(tokenTransferTxsBalance.plus(pendingTxsFeesBalance), 'STX');
+}
+
+function getPendingTxs(confirmedTxs: Transaction[], pendingTxs: MempoolTransaction[]) {
+  return pendingTxs.filter(tx => {
+    if (confirmedTxs.some(confirmedTx => confirmedTx.nonce === tx.nonce)) {
+      return false;
+    }
+    return tx.tx_type === 'token_transfer';
+  }) as unknown as MempoolTokenTransferTransaction[];
+}
+
+export function useMempoolTxsBalance(addresses: string[]) {
```

Comment:
I did some recreation to refactor the code we already had where `inbound` and `outbound` transactions separated. 

Here we use the same code for both but apply filtering to the `pendingTxs` in the same place to provide `inboundBalance` + `outboundBalance` for all supplied addresses